integration_tests/tests/test_logging/containerpilot.json5

```diff
@@ -0,0 +1,43 @@
+{
+  consul: "consul:8500",
+  logging: {
+    level: "DEBUG",
+    format: "text"
+  },
+  jobs: [
+    {
+      name: "job1",
+      // this echo gets wrapped in a log line
+      exec: "echo job1 exec",
+    },
+    {
+      name: "job2",
+      // this echo is not wrapped in a log line
+      exec: "echo job2 exec",
+      passthru: true,
```

Comment:
@misterbisson Unless the value was entirely unambiguous which, in this case, you'd still need to reference the docs for the non-bool value.